apps/openchallenges/db-cli/data/seeds/dev/organizations/challenge_organization.json

```diff
@@ -1 +1 @@
n":"uva","avatar_url":"https:\/\/via.placeholder.com\/300.png","website_url":"https:\/\/www.virginia.edu\/","description":"","challenge_count":1},{"id":"225","name":"University of Washington","email":"","login":"uw","avatar_url":"https:\/\/via.placeholder.com\/300.png","website_url":"https:\/\/www.washington.edu\/","description":"","challenge_count":7},{"id":"226","name":"University of Wisconsin-Madison","email":"","login":"uw-madison","avatar_url":"https:\/\/via.placeholder.com\/300.png","website_url":"https:\/\/www.wisc.edu\/","description":"","challenge_count":0},{"id":"227","name":"University of Zurich","email":"","login":"uzh","avatar_url":"https:\/\/via.placeholder.com\/300.png","website_url":"https:\/\/www.uzh.ch\/en.html","description":"","challenge_count":2},{"id":"228","name":"Urban Green Energy","email":"","login":"uge","avatar_url":"https:\/\/via.placeholder.com\/300.png","website_url":"https:\/\/www.ugei.com\/","description":"","challenge_count":1},{"id":"229","name":"US Army Medical Research Institute of Infectious Diseases","email":"","login":"usamriid","avatar_url":"https:\/\/via.placeholder.com\/300.png","website_url":"https:\/\/www.usamriid.army.mil\/","description":"","challenge_count":1},{"id":"230","name":"VA Durham Health Care","email":"","login":"va-durham-health-care","avatar_url":"https:\/\/via.placeholder.com\/300.png","website_url":"https:\/\/www.durham.va.gov\/ ","description":"","challenge_count":2},{"id":"231","name":"Verily","email":"","login":"verily","avatar_url":"https:\/\/via.placeholder.com\/300.png","website_url":"https:\/\/verily.com\/","description":"","challenge_count":1},{"id":"232","name":"VHA Innovation Ecosystem","email":"","login":"vha-ie","avatar_url":"https:\/\/via.placeholder.com\/300.png","website_url":"https:\/\/www.va.gov\/innovationecosystem\/","description":"","challenge_count":1},{"id":"233","name":"Washington University in St. Louis","email":"","login":"wustl","avatar_url":"https:\/\/via.placeholder.com\/300.png","website_url":"https:\/\/wustl.edu\/","description":"","challenge_count":1},{"id":"234","name":"Wayne State University","email":"","login":"wayne-state-university","avatar_url":"https:\/\/via.placeholder.com\/300.png","website_url":"https:\/\/wayne.edu\/","description":"","challenge_count":1},{"id":"235","name":"Weizmann Institute of Science","email":"","login":"weizmann-institute-of-science","avatar_url":"https:\/\/via.placeholder.com\/300.png","website_url":"https:\/\/www.weizmann.ac.il\/pages\/","description":"","challenge_count":2},{"id":"236","name":"Wellcome Sanger Institute","email":"","login":"sanger","avatar_url":"https:\/\/via.placeholder.com\/300.png","website_url":"https:\/\/www.sanger.ac.uk\/","description":"","challenge_count":2},{"id":"237","name":"White House Office of Science and Technology Policy","email":"","login":"ostp","avatar_url":"https:\/\/via.placeholder.com\/300.png","website_url":"https:\/\/www.whitehouse.gov\/ostp\/","description":"","challenge_count":1}]
```

Comment:
One improvement in the future would be to prettify this file and similar ones so that we can see what has changed on GitHub.

Reply:
Where's the fun in that? 😜 

But I agree -- I'll add that improvement now.